### PR TITLE
New Task: Exposing Pod Info ... Env Vars

### PR DESCRIPTION
**Pull request by steveperry-53**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2477)

### CODE REVIEW

File: docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+Next, get a shell into the Container that is running in your Pod:
+
+```
+kubectl exec -it dapi-envars-fieldref -- sh
+```
+
+In your shell, view the environment variables:
+
+```
+/ # printenv
```
Comment (chenopis) [inline docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md]: In other examples I've seen, the shell prompt is "/#" without the space. Which one is correct? Or does it not matter?

File: docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
+---
+title: Exposing Pod Information Through Environment Variables
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md]: This is a little bit vague on who the information is being exposed to.  WDYT of "Exposing Pod Information To Containers" or "Accessing Pod Information From Containers".

File: docs/tasks/configure-pod-container/dapi-envars-pod.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dapi-envars-fieldref
+spec:
+  containers:
+    - name: test-container
+      image: gcr.io/google_containers/busybox
+      command: [ "sh", "-c"]
+      args: ["while true; do echo -en '\n'; \
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/dapi-envars-pod.yaml]: Why not use the multiline yaml syntax.  I think it is something like this:

args: |
  while true; do
    echo -en '\n';
    printenv MY_NODE_NAME MY_POD_NAME MY_POD_NAMESPACE \
                   MY_POD_IP MY_POD_SERVICE_ACCOUNT;
    sleep 10;
    done

File: docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md (hunk 1 above)
Comment (steveperry-53) [inline docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md]: Fixed.

File: docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md (hunk 2 above)
Comment (steveperry-53) [inline docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md]: Done.

File: docs/tasks/configure-pod-container/dapi-envars-pod.yaml (hunk 3 above)
Comment (steveperry-53) [inline docs/tasks/configure-pod-container/dapi-envars-pod.yaml]: Done.

**Comment by steveperry-53** [comment]
@chenopis and @pwittrock, I addressed your comments. I don't know why the Travis build is failing now.

File: docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+1
+33554432
+67108864
+```
+
+{% endcapture %}
+
+{% capture whatsnext %}
+
+* [Exposing Pod Information Using a DownwardAPIVolumeFile](/docs/tasks/configure-pod-container/downward-api-volume-expose-pod-information/)
```
Comment (pwittrock) [inline docs/tasks/configure-pod-container/environment-variable-expose-pod-information.md]: This is a 404

**Comment by pwittrock** [comment]
Looks good.  Please fix the 404 before merging.

**Comment by steveperry-53** [comment]
I removed the broken link for now. After the DownwardAPIVolumeFile task gets published, I'll put the link back in this topic.

**Comment by steveperry-53** [comment]
@pwittrock, @jeffmendoza, Can you take a look at why the Travis CI build is failing.

**Comment by jeffmendoza** [comment]
Looking into it.

I don't mind

**Comment by jeffmendoza** [comment]
I have a travis fix here: #2489

**Comment by pwittrock** [comment]
It may have something to do with all the apiserver refactoring that has been going on.

**Pull request merged**